src/main/java/seedu/address/logic/commands/ClientNoteAddCommand.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+import seedu.address.model.note.Note;
+
+
+
+/**
+ * todo: add javadocs for ClientNoteAddCommand
+ */
+public class ClientNoteAddCommand extends Command {
+
+    public static final String COMMAND_WORD = "client note add";
+    private static final String MESSAGE_DUPLICATE_CLIENT_NOTE = "The client note already exists";
+    private static final String MESSAGE_SUCCESS = "Successfully added client note";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a note associated to a client "
+            + "by the index number used in the displayed client list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            // + "[" + PREFIX_TAG + "TAG]...\n"
+            + "NOTE_STRING"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + "client note content";
```

Comment:
Also, its hard to read when the lines are separated so much with the random comment also, would suggest this:

(must be a positive integer should also be moved to the description, not inside the parameters)
```suggestion
            + "Parameters: INDEX NOTE_STRING"
            + "Example: " + COMMAND_WORD + " 1 " + "client note content";
```

Reply:
yeap, done